newm-server/src/main/kotlin/io/newm/server/features/song/database/SongEntity.kt

```diff
@@ -24,7 +24,7 @@ import org.jetbrains.exposed.sql.or
 import org.jetbrains.exposed.sql.select
 import org.jetbrains.exposed.sql.selectAll
 import java.time.LocalDateTime
-import java.util.UUID
+import java.util.*
```

Comment:
This is caused because there is an IntelliJ option that causes to import a wildcard, even if you only import a single class of java.util.*.  I don't like that and I disabled. Let me know if you want to do the same and I can point you to the right place.

Reply:
Please point me to the right spot.